charts/scalardl/values.yaml

```diff
@@ -90,7 +92,8 @@ envoy:
     namespace: monitoring
 
   # envoy.resources -- resources allowed to the pod
-  resources: {}
+  resources:
+    {}
```

Comment:
Why do you add the line breaks? If I render a Chart template, it still contains them.

```
$ helm template charts/scalardl-audit | grep resources -A 2
          resources:
            {}
---
--
          resources:
            {}
```

As with many manifests, I feel it is more natural to not have line breaks.